.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -78,6 +78,7 @@ data class CIBuildModel(
             functionalTests = listOf(
                 TestCoverage(8, TestType.soak, Os.linux, JvmCategory.MAX_VERSION.version, vendor = JvmCategory.MAX_VERSION.vendor),
                 TestCoverage(9, TestType.soak, Os.windows, JvmCategory.MIN_VERSION.version, vendor = JvmCategory.MIN_VERSION.vendor),
+                TestCoverage(35, TestType.soak, Os.macos, JvmCategory.MIN_VERSION.version, vendor = JvmCategory.MIN_VERSION.vendor),
```

Comment:
What are these numbers? And shouldn't this entry come after `34`?

Reply:
The numbers need to be unique to identify a test type. So the URLs/UUIDs of the Teamcity configurations stay the same.`35` was the next free number, though I want to have this soak test as part of Ready for Release.